src/plugins/dashboard/public/application/top_nav/show_share_modal.tsx

```diff
@@ -151,19 +154,25 @@ export function ShowShareModal({
     ...unsavedStateForLocator,
   };
 
+  let _g = getStateFromKbnUrl<QueryState>('_g', window.location.href);
+  if (_g?.filters && _g.filters.length === 0) {
+    _g = _.omit(_g, 'filters');
```

Comment:
super tiny nit here, I like to import the various lodash functions rather than the whole library -  like:

`import { omit } from 'lodash'`.

It doesn't change the bundle size as far as I know, but it looks cleaner in usage.

Reply:
Done 👍